bodhi/server/services/releases.py

```diff
@@ -332,3 +337,75 @@ def save_release(request):
     request.db.flush()
 
     return r
+
+
+@critpath_packages.get(permission='edit', renderer='json',
+                       error_handler=bodhi.server.services.errors.json_handler,
+                       validators=(colander_body_validator, validate_release_name)
+                       )
```

Comment:
My personal preference would be to include this on the line above it to save precious monitor real estate, but this is fine if you prefer it as long as flake8 is happy.